src/plugins/blocksmith/utils/insertEmptyBlock.js

```diff
@@ -20,29 +21,33 @@ import normalizeKey from './normalizeKey';
  *
```

Comment:
@arunkarnati ? it will be a string or a Symbol depending on where we land with that. changed to Symbol for now

Reply:
@efjacobson - sorry, it is the `key` parameter I was gonna point out, check this out: 
https://github.com/triniti/cms-js/blob/atomic-convert-to-text/src/plugins/blocksmith/utils/deleteSelectedBlocks.js#L50

we are passing a `number`

Reply:
@arunkarnati aha - i see the confusion. it can actually be a number, string, or block as it is is used for `findBlock` https://github.com/triniti/cms-js/blob/master/src/plugins/blocksmith/utils/findBlock.js#L8

updated